### PR TITLE
Fix: assets with embedded image was overriding non embedded asset 

### PR DESCRIPTION
**Pull request by oa495**
- Now we **either** distribute the embedded image asset or distribute the non embedded image asset.

### CODE REVIEW

File: packages/@icon-magic/distribute/src/distribute-svg.ts [hunk 1]
```diff
@@ -187,6 +187,10 @@ function getAssetsToDistribute(icon: Icon, type: FlavorType, colorScheme: string
     assetsToDistribute = assetsToDistribute.filter(asset => {
       return asset.name.match(/-with-image/) ? true : false;
     });
+  } else {
+    assetsToDistribute = assetsToDistribute.filter(asset => {
+      return asset.name.match(/-with-image/) ? true : false;
+    });
```
Comment (camario25) [inline packages/@icon-magic/distribute/src/distribute-svg.ts:193]: Am I missing something but if you put an else statement here to the if statement above, and they are the same on the inside.. don't you just need to it without the if statement?  Isn't it going to happen no matter what?

**Review by camario25 — COMMENTED** [review]
qq in the comments, I might be missing something. Ty

File: packages/@icon-magic/distribute/src/distribute-svg.ts [hunk 2]
```diff
@@ -187,6 +187,10 @@ function getAssetsToDistribute(icon: Icon, type: FlavorType, colorScheme: string
     assetsToDistribute = assetsToDistribute.filter(asset => {
       return asset.name.match(/-with-image/) ? true : false;
     });
+  } else {
+    assetsToDistribute = assetsToDistribute.filter(asset => {
+      return asset.name.match(/-with-image/) ? true : false;
```
Comment (oa495) [inline packages/@icon-magic/distribute/src/distribute-svg.ts:192]:
```suggestion
      return asset.name.match(/-with-image/) ? false : true;
```

File: packages/@icon-magic/distribute/src/distribute-svg.ts (hunk 1 above)
Comment (oa495) [reply packages/@icon-magic/distribute/src/distribute-svg.ts]: oh oops updated

**Review by camario25 — APPROVED** [review]
got it, Ty!

**Pull request merged**